beacon-chain/rpc/aggregator/server.go

```diff
@@ -79,24 +78,8 @@ func (as *Server) SubmitAggregateAndProof(ctx context.Context, req *pb.Aggregati
 	// Retrieve the unaggregated attestation from pool
 	atts := as.AttPool.UnaggregatedAttestationsBySlotIndex(req.Slot, req.CommitteeIndex)
 
-	headState, err := as.HeadFetcher.HeadState(ctx)
-	if err != nil {
```

Comment:
On insertion into the pool, the attestations are verified, correct ?

Reply:
only the signature is verified, the attestation is fully verified when it is accounting for fork choice and getting packaged into a beacon block